src/render.rs

```diff
@@ -0,0 +1,53 @@
+/// `fpm::render()` renders a single ftd file that is part of current FPM package.
```

Comment:
Update documentation PR to mention that `fpm::render()` is implemented in a file `src/render.rs` as an implementation note.